content/tracing/services/service.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Service
+kind: Documentation
+further_reading:
+- link: "/tracing/services/resource"
+  text: "Dive into your resource performances and traces"
+- link: "/tracing/services/trace"
+  text: "Understand how to read a Datadog Trace"
+---
+
+Selecting a service on the services page leads you to the detailed service page:
+
+{{< img src="tracing/services/service/detailed_service_page.png" alt="Detailed service page" responsive="true" popup="true" style="width:90%;">}}
+
+Consult on this page:
+
+* Your [service monitor states](#service-monitor) 
+* Your [out of the box graphs](#out-of-the-box-graphs) 
+* Your [Resources](/tracing/services/resource) 
+
+## Definition
+
+**A Service is the name of a set of processes that do the same job.**  
```

Comment:
"A service is a set of processes that do the same job."

Reply:
👌